templates/web/fixmystreet.com/about/open311-api-info.html

```diff
@@ -0,0 +1,154 @@
+
+<dt>What if we can’t implement this ourselves due to time or resource constraints?</dt>
+<dd>We can build and maintain the integration for you (carries an annual fee). Please contact us if you’d like to discuss this at support@fixmystreet.com.</dd> 
+
+<dt>Can we integrate with FixMyStreet in a different way?</dt>
+<dd>Yes, you can, but Open311 is a standardised way of sharing information, and provides the foundations should you ever want to use FixMyStreet Pro in the future.</dd> 
+</dl>
+
+<dt>Limitations</dt>
+
+<dd>
+<p>Open311 fetches a list of acceptable types of report as dictated by you, the council, so that the receiving service (in this case, FixMyStreet) can display them as options for the report-maker to choose from, sends reports into a backend system of your choice and enables you to close or update reports accordingly.</p>
+<p>What it doesn’t do is the next steps that may be required to further refine the reporting process, reduce unnecessary contact or help triage reports. This is where you might want to consider using FixMyStreet Pro - a more advanced version of FixMyStreet, which seeks to eliminate the significant costs councils incur from duplicate reporting, unnecessary contact and failure demand.</p>
+
+<table>
+  <tr>
+    <th>FixMyStreet features</th>
+    <th>Open311</th>
+    <th>FixMyStreet Pro</th>
+  </tr>
+ <tr>
+    <td>Send reports into backend systems</td>
+    <td align="center" style="text-align:center; font-size:150%; font-weight:bold; color:green;">&#x2714</td>
+    <td align="center" style="text-align:center; font-size:150%; font-weight:bold; color:green;">&#x2714</td>
+  </tr>
+<tr>
+    <td>Close/update reports</td>
+    <td align="center" style="text-align:center; font-size:150%; font-weight:bold; color:green;">&#x2714</td>
+     <td align="center" style="text-align:center; font-size:150%; font-weight:bold; color:green;">&#x2714</td>
+  </tr>
+<tr>
+    <td>Categories dictated by your backend systems</td>
+    <td align="center" style="text-align:center; font-size:150%; font-weight:bold; color:green;">&#x2714</td>
+     <td align="center" style="text-align:center; font-size:150%; font-weight:bold; color:green;">&#x2714</td>
+  </tr>
+<tr>
+    <td>Extra questions</td>
+    <td align="center" style="text-align:center; font-size:150%; font-weight:bold; color:green;">&#x2714 *</td>
+     <td align="center" style="text-align:center; font-size:150%; font-weight:bold; color:green;">&#x2714</td>
+  </tr>
+<tr>
+    <td>Private/unadopted land</td>
+    <td align="center" style="text-align:center; font-size:150%; font-weight:bold; color:green;">&#x2714 **</td>
+     <td align="center" style="text-align:center; font-size:150%; font-weight:bold; color:green;">&#x2714</td>
+  </tr>
+ <tr>
+    <td>Duplicate report suggestions</td>
+    <td align="center" style="text-align:center; font-size:150%; font-weight:bold; color:green;">&#10008;</td>
```

Comment:
Do you want the crosses to be in green as well?

Reply:
Oh, no. I've changed to red...assuming that's possible